includes/data-events/README.md

```diff
@@ -0,0 +1,162 @@
+# Newspack Data Events
+
+## A tool for non-blocking dispatch of data events to registered handlers.
+
+The purpose of this tool is to unify the strategy for sending reader activity data to third-party services, focused primarily for analytics, ESPs and CRMs.
+
+The non-blocking strategy is inspired by [TechCrunch's `wp-async-task`](https://github.com/techcrunch/wp-async-task) and uses an HTTP request to trigger the handlers.
+
+---
+
+## Core Actions
```

Comment:
Maybe call it "Newspack core" action so we avoid confusion with WP core's actions

Reply:
Add: These events are registered by the Newspack Plugin and present by default

Reply:
428e0663305a143bc88ef92f46933035ae852ff7 updates this section